### ISSUE TITLE
change username

### ISSUE THREAD

**Issue by ltalirz** [marvel-nccr/marvel-virtualmachine#16]
change username from `vagrant` to something reasonable (e.g. `marvel`).

This requires adding a new user in a very first provisioning step before ansible sets in (ansible needs to use the final username).

reporter: Nicola Marzari

**Comment by ltalirz** [comment]
this might help http://willhaley.com/blog/different-ssh-users-for-vagrant-ssh-and-vagrant-up/

**Comment by ltalirz** [comment]
Summing up some extensive trial and error:

Assuming that we will continue to use the `bento` base images with pre-configured vagrant user, there are two possible approaches:
 1. Let `vagrant` deal with two usernames
 2. Let `ansible` deal with two usernames

I've played around with 2. as it seemed easier at first, but in the end it doesn't work (see branch `issue_16_change_username`

The problem with 1. is that vagrant doesn't seem to have the concept of using different ssh users for different provisioning steps (which would be all we need: the first provisioner sets up the new user, and from then on we proceed as usual with the new user name).
One possible way out may be to use two separate Vagrant files.
